module/Decision/src/Controller/DecisionController.php

```diff
@@ -155,10 +160,16 @@ public function searchAction(): ViewModel
         );
     }
 
-    public function authorizationsAction(): ViewModel
+    public function authorizationsAction(): Response|ViewModel
     {
         if (!$this->aclService->isAllowed('create', 'authorization')) {
-            throw new NotAllowedException($this->translator->translate('You are not allowed to authorize someone'));
+            $this->flashMessenger()->addErrorMessage($this->translator
+                ->translate('You are not allowed to authorize someone'));
```

Comment:
Please remove this such that we can assert that the `throw` is the only thing that will happen here.